### ISSUE TITLE
Search always return false

### ISSUE THREAD

**Issue by Albvadi** [ecrmnn/collect.js#242]
Hi,
I have problems searching elements.

```js
    const collection = collect([
      { id: 1, name: "Test" },
      { id: 2, name: "Test2" }
    ]);

    console.log(collection.search((item, key) => item.id > 1));

// EXPECTS: 1
// RETURNS: false
```

I´m a bit newbie.. Sorry if it´s a stupid issue... :disappointed_relieved:

**Comment by ecrmnn** [comment]
I'll have a look.

I would recommend to use `where`
```js
collection.where('id', '>', 1);
```

**Comment by dtvmedia** [comment]
Why is this closed? I just stumbled upon a similar scenario and had the same results. The key is expected to be returned here. Your suggestion of using `where` will return the corresponding item instead.

**Comment by ecrmnn** [comment]
You are right @dtvmedia.

Fixed in `4.25.0`
https://github.com/ecrmnn/collect.js/commit/faaa1983699a6ca9db62e6bfc2f74da0b1726ac8)
